nimble/host/syscfg.yml

```diff
@@ -110,6 +105,11 @@ syscfg.defs:
             Defines maximum number of LE Connection Oriented Channels channels.
             When set to (0), LE COC is not compiled in.
         value: 0
+    BLE_L2CAP_MAX_CHANS:
+        description: >
+            The number of L2CAP channels to allocate.  The default value allows
+            for the signal, ATT, and SM channels for each connection.
```

Comment:
Why is not valid?
BLE_L2CAP_MAX_CHANS describes the maximum l2cap channels supported per connection and it has a default value of 3 which are signal, Att and SM which are fixed channels.
also BLE_L2CAP_COC_MAX_NUM is still valid because it describes the maximum number of supported l2cap coc channels.

Reply:
That's correct. maybe it needs more description . I will modify it.